### DISCUSSION TITLE
Sonarr and Radarr - Add Root Folder - NFS

### DISCUSSION THREAD

**Discussion by gg-coder-dev** [tteck/Proxmox#1666] [Help Needed]
Hello!
I am beginner at linux, self hosting so if you can, talk like to talk to child.

I installed Jellyfin from script. Mounted my Nas disk using commands: 
`sudo apt update
sudo apt install cifs-utils
sudo mkdir -p /mnt/nas
sudo nano /etc/fstab
//192.168.1.1/media   /mnt/nas	cifs	user=gniewko,password=mypassword,uid=1000,gid=1000,		0	0
mount //192.168.1.1/media
`

and it worked. I tried do the same on Sonarr and Radarr but without success. 
First error: mount error(1): Operation not permitted
Refer to the mount.cifs(8) manual page (e.g. man mount.cifs) and kernel log messages (dmesg)

Next i tried to change uid for 101001, than another error, then i read sth about  Privileged container but looks like my knowledge is too low to acomplish this.
Could you give me solution, commands? 
Thanks you in advance :)

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/discussions/1316

**Reply by gg-coder-dev** [reply]
I used that command and still cant mount. Still the same error
